### ISSUE TITLE
Autorandr postswitch triggered when chrome/brave is started

### ISSUE THREAD

**Issue by DawidJanczak** [phillipberndt/autorandr#310]
Hello,

Not sure if this is an issue with autorandr or with chrome, but whenever I start chrome or brave from command line (e.g. using `xdg-open`) it triggers autorandr execution hook. Any idea if this can be avoided in any way? Logs from journalctl:

```
Nov 10 12:57:10 hp kernel: [drm] PCIE GART of 256M enabled (table at 0x000000F400000000).
Nov 10 12:57:10 hp kernel: [drm] UVD and UVD ENC initialized successfully.
Nov 10 12:57:10 hp kernel: [drm] VCE initialized successfully.
Nov 10 12:57:10 hp systemd[1]: Starting autorandr execution hook...
░░ Subject: A start job for unit autorandr.service has begun execution
░░ Defined-By: systemd
░░ Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
░░
░░ A start job for unit autorandr.service has begun execution.
░░
░░ The job identifier is 2916.
Nov 10 12:57:12 hp autorandr[8369]: [{"success":true}]
Nov 10 12:57:12 hp autorandr[8346]: autorandr running as user *** (started from batch instance)
Nov 10 12:57:12 hp autorandr[8346]: docked (detected)
Nov 10 12:57:12 hp autorandr[8346]: mobile
Nov 10 12:57:12 hp autorandr[8346]: tv
Nov 10 12:57:12 hp autorandr[8339]: Running autorandr as *** for display :0
Nov 10 12:57:12 hp systemd[1]: autorandr.service: Deactivated successfully.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
░░
░░ The unit autorandr.service has successfully entered the 'dead' state.

```

**Comment by phillipberndt** [comment]
Fix causes #324

**Comment by DawidJanczak** [comment]
Just in case someone runs into this: this was only happening to me with discrete graphics card enabled. I didn't really use it and after disabling it and only using the integrated graphics I no longer see this behaviour.

This of course doesn't fix the underlying issue, but could be a pointer for further investigation if someone wishes to do so.
